runtime/runtime-core/common/test/aws/smithy/kotlin/runtime/retries/impl/StandardRetryPolicyTest.kt

```diff
@@ -38,7 +38,7 @@ class StandardRetryPolicyTest {
     }
 
     @Test
-    fun testClientSideException() {
+    fun testRetryableClientException() {
```

Comment:
**Question**: Why did this name change? It's not technically testing for a `ClientException` where `isRetryable` is true...it's still testing a `ServiceException` with `errorType == ErrorType.Client`.

Reply:
Oh you're right I made the update here quickly thinking htis was testing a retryable client exception and missed the fact that this is a service side exception with error type of client (confused yet?). I'll update the name to something like `testClientErrorType` or something.